package/ports_daemon/ports_daemon/src/ports.c

```diff
@@ -237,7 +237,7 @@ static port_config_t port_configs[] = {
     .restart = RESTART,
     .first_start = true,
     .blacklist_len = 1,
```

Comment:
This is the danger that I worried about when I wrote the blacklist this way: all these `blacklist_len` need to be manually updated, and we need to manually remember every single time we change this...

Reply:
Repeat "update blacklist_len" comment for all further occurences as well

Reply:
now this is a useful catch. I definitely didn't think about it when page-wise search and replacing